core/src/main/java/com/github/kaktushose/jda/commands/dispatching/reply/internal/ReplyAction.java

```
@@ -123,9 +124,7 @@ public Message reply() {
                     deferEdit(modalEvent);
             case IMessageEditCallback callback when editReply -> deferEdit(callback);
             case IReplyCallback callback -> deferReply(callback);
-            default -> throw new IllegalArgumentException(
-                    "Cannot reply to '%s'! Please report this error to the devs of jda-commands!".formatted(event.getClass().getName())
-            );
+            default -> throw new JDACException.Internal("Cannot reply to '%s'! Please report this error to the devs of jda-commands!", event.getClass().getName());
```

"Please report this error to the devs of jda-commands." can be removed, would be duplicated